src/psyclone/psyGen.py

```diff
@@ -1465,7 +1465,9 @@ def data_on_device(self, parent):
 class ACCParallelDirective(ACCDirective):
     '''
     Class representing the !$ACC PARALLEL directive of OpenACC
-    in the PSyIR.
+    in the PSyIR. By default it includes the 'DEFAULT(PRESENT)' clause which
```

Comment:
Several AccDirective classes have the pylint issue  "Method '__str__' is abstract in class 'Node' but is not overridden", but the `node.py:node.__str__` should not be abstract, could you remove the `@abc.abstractmethod` in that method.

Reply:
Yep, done.